Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV DATABASE_URL=postgresql://root@localhost/vmdb_production?encoding=utf8&pool=
 
 RUN yum -y install --setopt=tsflags=nodocs \
                    memcached               \
-                   postgresql-server       \
+                   postgresql-server	   \
```

Comment:
Can you fix the spacing so that this diff is removed from the patch?

Reply:
yes, done